packages/dx-react-scheduler/src/types/view-switcher/view-switcher.types.ts

```diff
@@ -1,14 +1,19 @@
+import { SchedulerView } from '@devexpress/dx-scheduler-core';
```

Comment:
We should import types from `dx-scheduler-core` into `index.ts` file. And here we should use type from `index.ts` file.

See [here](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-scheduler/src/types/index.ts#L3).